doc/src/modules/physics/mechanics/masses.rst

```diff
@@ -352,9 +356,9 @@ The user can then determine the kinetic energy of any number of entities of the
 system: ::
 
   >>> kinetic_energy(N, Pa)
-  2*l1**2*m*omega(t)**2
```

Comment:
Look at the lines above - previously the printing was inconsistent, sometime with `(t)` and sometimes without. This makes it all the same.

Removing `init_vprinting` would do the reverse and use `(t)` everywhere - but at that point, there'd be no reason to use `dynamicsymbols` in this example.